labelbox/schema/dataset.py

```diff
@@ -135,7 +142,10 @@ def convert_item(item):
             }
 
         # Prepare and upload the desciptor file
-        data = json.dumps([convert_item(item) for item in items])
+        if not tiled_imagery:
```

Comment:
Is there no way that we can detect tiled imagery and do the conversion ourselves if necessary? ( rather than the user explicitly telling us)

Reply:
Looking into it.

Reply:
fixed.